### ISSUE TITLE
10 Squares! - I thought of a problem

### ISSUE THREAD

**Issue by coke11** [csf30816/scratchviewer#3]
Alright. You know how you were going to add love and favorite buttons? The problem with that, would be that if a project is unshared, you could still like it, and people would think you were going around hacking when they saw you were liking a whole bunch of unshared projects.

http://scratch.mit.edu/projects/128151649/
http://scratchviewer.usa.cc/#128151649
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.87 Safari/537.36

**Comment by csf30816** [comment]
I thought of that too... I might have an answer but it will delay the love/fave thing till 2017

**Comment by coke11** [comment]
What is the answer?

**Comment by csf30816** [comment]
Only displaying the love/fave buttons if the projects is registered as shared on Scratch

**Comment by joker314** [comment]
How can you get it to love/like projects? What account will do the liking? You can't exactly access the user's account very easily

**Comment by csf30816** [comment]
I do not know anymore. It has takin me 10  months to think and still do not know.

**Comment by csf30816** [comment]
The love/fave thing (it think) is impossible. Still trying to find out how it may work.

**Comment by AmazingMech2418** [comment]
I've made the love/fave thing before. Also, you can only love/fave your own unshared projects.

**Comment by csf30816** [comment]
Yes, hi @AmzingMech2418 I already know how to do it now. But I am not working on this anymore. Since now Phosphorus is back in production. I made this because it was not but it is now :)
